global/get-modifed-folders/action.yml

```diff
@@ -0,0 +1,196 @@
+name: "📁 Get modified directory and sparse checkout"
+description: "Retrieve all modified directory and perform a sparse checkout."
+
+
+inputs:
+  starting_index:
+    description: '(Optional) Starting index for path filtering'
+    default: '/'
+  default_end_index:
```

Comment:
@ffppa since the term "index" is used to identify the starting folder, i suggest using "default_end_folder_depth" o something similar as the name for this variable to avoid ambiguity

Reply:
suggesting appliend on [PR-40](https://github.com/pagopa/eng-github-actions-iac-template/pull/40)